PhysicsTools/NanoAOD/plugins/JetSorter.cc

```diff
@@ -0,0 +1,53 @@
+#include "FWCore/Framework/interface/stream/EDProducer.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+
+#include "DataFormats/Common/interface/View.h"
+#include "DataFormats/PatCandidates/interface/Jet.h"
+
+class JetSorter : public edm::stream::EDProducer<> {
```

Comment:
could be `edm::global::EDProducer<>`; member variables are not modified during `produce()`